google/cloud/storage/blob.py

```diff
@@ -2507,7 +2514,7 @@ def _prep_and_do_upload(
         if_metageneration_not_match=None,
         timeout=_DEFAULT_TIMEOUT,
         checksum="auto",
-        retry=DEFAULT_RETRY_IF_GENERATION_SPECIFIED,
+        retry=DEFAULT_RETRY,
```

Comment:
lgtm, curious if we want to update the default retry for internal methods `_do_upload()` `_do_multipart_upload()` `_do_resumable_upload()`

I see it doesn't really matter as `_prep_and_do_upload` does most of the upload logic handling after the previous refactoring, so this is just a question out of curiousity

Reply:
Yes, this was an oversight, I'll fix it. Thanks for noticing.